packages/components/src/private-apis.ts

```diff
@@ -10,13 +10,16 @@ import { Tabs } from './tabs';
 import { kebabCase } from './utils/strings';
 import { lock } from './lock-unlock';
 
-export const privateApis = {};
-lock( privateApis, {
+const privateData = {
 	__experimentalPopoverLegacyPositionToPlacement,
 	createPrivateSlotFill,
 	ComponentsContext,
 	Tabs,
 	Theme,
 	Menu,
 	kebabCase,
-} );
+};
+
+export const privateApis = {} as typeof privateData;
+
+lock( privateApis, privateData );
```

Comment:
Is is only to set the proper type for `privateApis` here to ensure the consumer components that use `lock( privateApis )` receive proper types.